tests/testthat/test-get_contact_matrix.R

```diff
@@ -0,0 +1,11 @@
+#testing for the get_contact_matrix.R function
+test_that("get_contact_matrix produces expected 21x21 square matrix output", {
+
+  #run the function with Great Britain and contact_home
+  home_matrix <- get_contact_matrix("United Kingdom of Great Britain",
+                                    contact_home, 21)
+
+  #expect the length of both dimensions of the output to be equal to 21.
+  expect_length(home_matrix[1, ], 21)
+  expect_length(home_matrix[, 1], 21)
```

Comment:
Is it worth having a check that the contact counts in 80+ years are the same as at 80? I.e. just checking that the value copying has worked as expected. This will be easier if we have named rows and columns.

Reply:
Good Idea, I've written a test that checks that the function has copied the contacts correctly

Reply:
Coolio.